ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -95,6 +95,11 @@ open class CommandBar: UIView {
         trailingCommandGroupsView.updateButtonsState()
     }
 
+    /// Sets the scoll position  to the start of the scroll view
+    @objc public func resetScrollPosition() {
+        scrollView.scrollRectToVisible(CGRect(x: 0, y: 0, width: 1, height: 1), animated: false)
```

Comment:
Does CGRectZero (.zero?) not work? If not, should at least add a comment why we need the magic 1x1 sized rect.

Reply:
`CGRectZero` does not work. The `width` and `height` need to greater than 0 in order the scrolling to occur.